### ISSUE TITLE
Dark native  theme  dont work 

### ISSUE THREAD

**Issue by wolfkingal2000** [Lymphatus/caesium-image-compressor#316]
Dark native  theme  dont work 

vr=2.8.2

**Comment by Lymphatus** [comment]
Are you using Windows 10? Native dark theme on Windows 10 may not work ion it, you need to use Fusion.

**Comment by wolfkingal2000** [comment]
> Are you using Windows 10? Native dark theme on Windows 10 may not work ion it, you need to use Fusion.


Yes, I'm using Windows 10, but the fusion theme is very ugly. I want it to be completely black like the previous versions.

**Comment by 7eventech77** [comment]
> > Are you using Windows 10? Native dark theme on Windows 10 may not work ion it, you need to use Fusion.
> 
> Yes, I'm using Windows 10, but the fusion theme is very ugly. I want it to be completely black like the previous versions.

I do agree 2.8's fusion dark theme is a little weird compared 2.7 and older

**Comment by Lymphatus** [comment]
Prior to 2.8.x I was using a custom palette of colors for the dark Fusion theme, while the one you see now is the "original". It's not a big deal to revert it. I will evaluate it.

**Comment by Lymphatus** [comment]
Fixed in [v2.8.3](https://github.com/Lymphatus/caesium-image-compressor/releases/tag/v2.8.3)
